x-pack/solutions/security/plugins/security_solution/server/endpoint/services/actions/clients/sentinelone/sentinel_one_actions_client.ts

```diff
@@ -124,6 +127,120 @@ export class SentinelOneActionsClient extends ResponseActionsClientImpl {
     connectorActions.setup(SENTINELONE_CONNECTOR_ID);
   }
 
+  private async fetchSentinelOneAgentIndexNames(): Promise<string[]> {
+    const cachedInfo = this.cache.get<string[]>('fetchSentinelOneAgentIndexNames');
+
+    if (cachedInfo) {
+      this.log.debug(
+        `Returning cached response with list of Agent index names:\n${stringify(cachedInfo)}`
+      );
+      return cachedInfo;
+    }
+
+    const fleetServices = this.options.endpointService.getInternalFleetServices(
+      this.options.spaceId
+    );
+    const indexNamespaces = await fleetServices.getIntegrationNamespaces(['sentinel_one']);
+    const indexNames: string[] = [];
+
+    for (const [, namespaces] of Object.entries(indexNamespaces)) {
```

Comment:
```suggestion
    for (const namespaces of Object.values(indexNamespaces)) {
```

Reply:
and now I'm thinking: why did I do it this way? 😆 
will refactor to use `Object.values()` - thanks for the feedback.